sorts/topological_sort.py

```diff
@@ -8,28 +8,39 @@
 edges = {"a": ["c", "b"], "b": ["d", "e"], "c": [], "d": [], "e": []}
 vertices = ["a", "b", "c", "d", "e"]
 
+sort = []
+visited = {}
+
 
 def topological_sort(start, visited, sort):
-    """Perform topolical sort on a directed acyclic graph."""
+    """Perform topological sort on a directed acyclic graph."""
     current = start
     # add current to visited
-    visited.append(current)
+    visited[current] = 1
     neighbors = edges[current]
     for neighbor in neighbors:
         # if neighbor not in visited, visit
-        if neighbor not in visited:
+        if visited[neighbor] == 0:
             sort = topological_sort(neighbor, visited, sort)
     # if all neighbors visited add current to sort
-    sort.append(current)
-    # if all vertices haven't been visited select a new one to visit
-    if len(visited) != len(vertices):
-        for vertice in vertices:
-            if vertice not in visited:
-                sort = topological_sort(vertice, visited, sort)
+    sort.append(current)    
     # return sort
     return sort
 
+def print_graph(sort):
+    #reverse the sort 
+    sort = sort[::-1]
+    print(sort)
```

Comment:
Try to avoid including print statements

Reply:
Changes are made!
Can u please review it!